docs/developer-guide/local_testing_guide.md

```diff
@@ -0,0 +1,285 @@
+Aug 17 14:46:42.842: INFO: Running AfterSuite actions on all nodes
+
+
+Aug 17 14:46:15.264: INFO: Running AfterSuite actions on all nodes
+Aug 17 14:46:42.885: INFO: Running AfterSuite actions on node 1
+Aug 17 14:46:42.885: INFO: Skipping dumping logs from cluster
+
+
+Ran 1 of 5667 Specs in 30.921 seconds
+SUCCESS! -- 1 Passed | 0 Failed | 0 Flaked | 0 Pending | 5666 Skipped
+
+
+Ginkgo ran 1 suite in 38.489055861s
+Test Suite Passed
+~~~
+
+### Running a control-plane test
+
+All local tests are defined as `control-plane` tests. To run a single `control-plane` test, target the `control-plane`
+action and append the `WHAT=<test name>` parameter, as follows:
+
+```
+$ cd $REPO
+$ pushd test
+$ make control-plane WHAT="should be able to send multicast UDP traffic between nodes"
+$ popd
+```
+
+The value of `WHAT=` will be used to modify the `-ginkgo.focus` parameter. Individual tests can be retrieved from this
+repository under [test/e2e](https://github.com/ovn-org/ovn-kubernetes/tree/master/test/e2e). To see a list of individual
+tests, one could run:
+~~~
+grep -R ginkgo.It test/
+~~~
+
+For example:
+~~~
+# grep -R ginkgo.It . | head -1
+./e2e/multicast.go:	ginkgo.It("should be able to send multicast UDP traffic between nodes", func() {
+# make control-plane WHAT="should be able to send multicast UDP traffic between nodes"
+(...)
++ go test -timeout=0 -v . -ginkgo.v -ginkgo.focus 'should\sbe\sable\sto\ssend\smulticast\sUDP\straffic\sbetween\snodes' -ginkgo.flakeAttempts 2 '-ginkgo.skip=recovering from deleting db files while maintain connectivity|Should validate connectivity before and after deleting all the db-pods at once in HA mode|Should be allowed to node local cluster-networked endpoints by nodeport services with externalTrafficPolicy=local|e2e ingress to host-networked pods traffic validation|host to host-networked pods traffic validation' -provider skeleton -kubeconfig /root/ovn.conf --num-nodes=2 --report-dir=/root/ovn-kubernetes/test/_artifacts --report-prefix=control-plane_
+I0817 15:26:21.762483 1197731 test_context.go:457] Tolerating taints "node-role.kubernetes.io/control-plane" when considering if nodes are ready
+=== RUN   TestE2e
+I0817 15:26:21.762635 1197731 e2e_suite_test.go:67] Saving reports to /root/ovn-kubernetes/test/_artifacts
+Running Suite: E2e Suite
+(...)
+• [SLOW TEST:12.332 seconds]
+Multicast
+/root/ovn-kubernetes/test/e2e/multicast.go:25
+  should be able to send multicast UDP traffic between nodes
+  /root/ovn-kubernetes/test/e2e/multicast.go:75
+------------------------------
+SSSSSSSSSSSS
+JUnit report was created: /root/ovn-kubernetes/test/_artifacts/junit_control-plane_01.xml
+
+Ran 1 of 60 Specs in 12.333 seconds
+SUCCESS! -- 1 Passed | 0 Failed | 0 Flaked | 0 Pending | 59 Skipped
+--- PASS: TestE2e (12.34s)
+PASS
+ok  	github.com/ovn-org/ovn-kubernetes/test/e2e	12.371s
++ popd
+~/ovn-kubernetes/test
+~~~
+
+## IPv6 tests
+
+To skip the IPv4 only tests (in a IPv6 only deployment), pass the
```

Comment:
I thought we look this up now and dont rely on the env var for lookup of IP family support

Reply:
Remove the need to set the platforum support env var `PLATFORM_IPV6_SUPPORT` - the e2es lookup the supported IP families now.

Reply:
Sorry for asking you to do this since youre just moving this :P